eruditorg/templates/public/home.html

```diff
@@ -150,6 +150,7 @@ <h3 class="latest-issues--article--journal-title">
   {% endif %}
 
   <!-- NEWS -->
+  {% if latest_news %}
```

Comment:
@sssoz no `latest_news` probably means that "À propos" is down. In that case I suggest we hide the whole news container.